### PR TITLE
Update release.md

### PR DESCRIPTION
**Pull request by offtherailz**
## Description
Update of release procedure with snapshot

### CODE REVIEW

File: docs/developer-guide/release.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ Replacing:
     - [ ] Commit on master 0.&lt;x-incremented&gt;.0
     - [ ] Make sure on YYYY.XX.xx we have the 0.x.&lt;number-of-minor-version&gt;
 - [ ] Update `CHANGELOG.md`. [Instructions](https://mapstore.readthedocs.io/en/latest/developer-guide/release/#changelog-generation) - both master and stable
-- [ ] Fix `pom.xml` to stable versions ( no `-SNAPSHOT` in release).
+- [ ] Fix `pom.xml` dependencies stable versions ( no `-SNAPSHOT` usage release).
+- [ ] Update the version of java modules on the release branch to a stable, incremental version. (use mvn version commands)
```
Comment (offtherailz) [inline docs/developer-guide/release.md:41]:
```suggestion
- [ ] Update the version of java modules on the release branch to a stable, incremental version.
    - [ ] Run `mvn versions:set -DnewVersion=<VERSION> -DprocessAllModules -DgenerateBackupPoms=false` to update package version, where <VERSION> is the version of the java packages (e.g. `1..2.2`).
    - [ ] Manually update project pom templates to use `mapstore-services` of `<VERSION>`
```

File: docs/developer-guide/release.md [hunk 2]
```diff
@@ -37,7 +37,10 @@ Replacing:
     - [ ] Commit on master 0.&lt;x-incremented&gt;.0
     - [ ] Make sure on YYYY.XX.xx we have the 0.x.&lt;number-of-minor-version&gt;
 - [ ] Update `CHANGELOG.md`. [Instructions](https://mapstore.readthedocs.io/en/latest/developer-guide/release/#changelog-generation) - both master and stable
-- [ ] Fix `pom.xml` to stable versions ( no `-SNAPSHOT` in release).
+- [ ] Fix `pom.xml` dependencies stable versions ( no `-SNAPSHOT` usage release).
+- [ ] Update the version of java modules on the release branch to a stable, incremental version.
+    - [ ] Run `mvn versions:set -DnewVersion=<VERSION> -DprocessAllModules -DgenerateBackupPoms=false` to update package version, where <VERSION> is the version of the java packages (e.g. `1..2.2`).
```
Comment (offtherailz) [inline docs/developer-guide/release.md:42]:
```suggestion
    - [ ] Run `mvn versions:set -DnewVersion=<VERSION> -DprocessAllModules -DgenerateBackupPoms=false` to update package version, where <VERSION> is the version of the java packages (e.g. `1.2.2`).
```

**Review by MV88 — CHANGES_REQUESTED** [review]
just one thing

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
@offtherailz please backport

**Pull request merged**